scss/nacara.scss

```diff
@@ -63,3 +63,18 @@ body {
 .icon+span {
     margin-left: .5rem;
 }
+
+// TODO: Mobile width, is there a variable for this?
+@media (max-width: 768px) {
```

Comment:
We can avoid this custom CSS by using `is-hidden-mobile` [doc](https://bulma.io/documentation/modifiers/responsive-helpers/#hide) directly on the TOC element